src/panels/config/lovelace/dashboards/dialog-lovelace-dashboard-detail.ts

```diff
@@ -283,7 +283,13 @@ export class DialogLovelaceDashboardDetail extends LitElement {
       }
       this.closeDialog();
     } catch (err: any) {
-      this._error = { base: err?.message || "Unknown error" };
+      const errorMessage = err?.message;
+      if (errorMessage.includes(". Got")) {
```

Comment:
This should be fixed in core, and so too prone for errors. Core should send a specific error code.